client/src/scripts/managers/inputManager.ts

```diff
@@ -8,6 +8,7 @@ import { ItemType, type ItemDefinition } from "@common/utils/objectDefinitions";
 import { Vec } from "@common/utils/vector";
 import $ from "jquery";
 import nipplejs, { type JoystickOutputData } from "nipplejs";
+import * as PIXI from "pixi.js";
```

Comment:
please do not import everything from pixijs

Reply:
Done, now it only imports ticker